CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+## 0.1.0
+
+**Release date:** 2020-12-10
+
+This is the first ever release of image-automation-controller and its
```

Comment:
```suggestion
This is the first prerelease of image-automation-controller and its
```

Reply:
It's not a prerelease, unless you want to go through 0.1.0-rc* first

Reply:
We mark all 0.x versions as prereleases in GitHub https://github.com/fluxcd/source-controller/releases

Reply:
Ah, everything is a prerelease at present (until 1.0?). OK, got it.